platform/android/export/export.cpp

```diff
@@ -744,6 +744,34 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	//Needs to match the method parameters of EditorExportSaveSharedObject
```

Comment:
These will already be taken care of by https://github.com/godotengine/godot/pull/39864 hopefully

Reply:
Please make the changes I'm suggesting in that PR.